leapp/cli/upgrade/__init__.py

```diff
@@ -240,12 +240,14 @@ def upgrade(args):
         workflow.load_answers(answerfile_path, userchoices_path)
         workflow.run(context=context, skip_phases_until=skip_phases_until, skip_dialogs=True)
 
+    logger.info("Answerfile will be created at %s", answerfile_path)
+    workflow.save_answers(answerfile_path, userchoices_path)
```

Comment:
if you have this here, then it should be also in preupgrade

Reply:
@vinzenz it is (https://github.com/oamg/leapp/blob/master/leapp/cli/upgrade/__init__.py#L291-L292)

Reply:
OK nevermind then, I didn't realize it was there already. I missed the context in the PR view ;-)

So then LGTM